src/test/java/de/tum/in/www1/artemis/lecture/AttachmentUnitsIntegrationTest.java

```diff
@@ -79,34 +80,21 @@ void testAll_InstructorNotInCourse_shouldReturnForbidden() throws Exception {
     @Test
     @WithMockUser(username = TEST_PREFIX + "instructor1", roles = "INSTRUCTOR")
     void splitLectureFile_asInstructor_shouldGetUnitsInformation() throws Exception {
-        var filePart = createLectureFile(true);
-
-        LectureUnitInformationDTO lectureUnitSplitInfo = request.postWithMultipartFile("/api/lectures/" + lecture1.getId() + "/process-units", null, "process-units", filePart,
-                LectureUnitInformationDTO.class, HttpStatus.OK);
-
-        assertThat(lectureUnitSplitInfo.units()).hasSize(2);
-        assertThat(lectureUnitSplitInfo.numberOfPages()).isEqualTo(20);
+        assertThat(request.postWithMultipartFile(lecture1.getId(), createLectureFile(true), OK).units()).hasSize(2);
+        assertThat(request.postWithMultipartFile(lecture1.getId(), createLectureFile(true), OK).numberOfPages()).isEqualTo(20);
```

Comment:
I think you should only have one request..postWithMultipartFile(...) and then use the return value and use it for assertion. You might want to leave it similar as it was before